packages/cli/src/models/config/ConfigManager.ts

```diff
@@ -38,6 +42,7 @@ const ConfigManager = {
       await ZWeb3.checkNetworkId(network.networkId);
       const txParams = {
         from: ZWeb3.toChecksumAddress(from || artifactDefaults.from || (await ZWeb3.defaultAccount())),
+        ...pickBy(pick(artifactDefaults, ['gas', 'gasPrice']), x => !isNil(x)),
```

Comment:
Personally, I'd have added `negate` as well, and replaced `x => !isNil(x)` by `negate(isNil)`, but maybe it's because I've been haskelling too much in my free time.

Reply:
Ah, I didn't know about `negate`. I updated #1231 to recommend `omitBy(..., isNil)`, but forgot to reflect the change PR.